src/docfx/restore/RestoreFile.cs

```diff
@@ -76,11 +76,11 @@ public static string GetRestoreFileName(string hash, EntityTagHeaderValue etag =
         /// Get ETag from restore file name
         /// </summary>
         /// <returns>ETag of the resource, null if server doesn't specify ETag</returns>
-        public static EntityTagHeaderValue GetEtag(string restoreFileName)
+        public static EntityTagHeaderValue GetEtag(string restorePath)
```

Comment:
`RestoreFile.Restore` calls `GetEtag`, you might want to update that as well: `var existingEtag = GetEtag(existingPath)`.